### PR TITLE
Add static pages

### PR DESCRIPTION
**Pull request by rofiuddin**

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    <%= t 'system_name' %>
+    <a href="https://www.facebook.com/FramgiaBangladesh" target="_blank">© Framgia Bangladesh 2015</a>
```
Comment (abdullahallmehedi) [inline app/views/layouts/_footer.html.erb]: i18n

File: Gemfile [hunk 2]
```diff
@@ -43,4 +44,4 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
```
Comment (Rownaknira) [inline Gemfile]: extra indent

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<% provide(:title, t(:about)) %>
+<h1><%= t 'about' %></h1>
```
Comment (abdullahallmehedi) [inline app/views/static_pages/about.html.erb]: use symbol pls

File: app/views/static_pages/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+<p>
+  <%= t 'system_name' %>
```
Comment (abdullahallmehedi) [inline app/views/static_pages/about.html.erb]: use symbol pls

File: app/views/static_pages/contact.html.erb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, t(:contact)) %>
+<h1><%= t 'contact' %></h1>
```
Comment (abdullahallmehedi) [inline app/views/static_pages/contact.html.erb]: use symbol pls

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<div class="center jumbotron">
+  <p><%= t "welcome_message" %> <br> <%= t "system_name" %></p>
```
Comment (abdullahallmehedi) [inline app/views/static_pages/home.html.erb]: use symbol pls

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<div class="center jumbotron">
+  <p><%= t (:welcome_message) %> <br> <%= t (:system_name) %></p>
```
Comment (abdullahallmehedi) [inline app/views/static_pages/home.html.erb]: no need to use ()

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,19 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (abdullahallmehedi) [inline app/views/layouts/application.html.erb]: ()

File: app/views/layouts/_footer.html.erb [hunk 9]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    <%= t :system_name %>
+    <a href="https://www.facebook.com/FramgiaBangladesh" target="_blank"><%= t :footer_text %></a>
```
Comment (abdullahallmehedi) [inline app/views/layouts/_footer.html.erb]: More than 80 characters

**Comment by abdullahallmehedi** [comment]
ok.

**Comment by Rownaknira** [comment]
ok

**Pull request merged**